web/shared-beans/src/test/java/io/deephaven/web/shared/data/RangeTest.java

```diff
@@ -45,4 +45,13 @@ public void testAdjacentMerge() {
         assertEquals(new Range(0, 5), rangeB.overlap(rangeA));
     }
 
+    @Test
+    public void testLargeRange() {
+        long largeA = 16535734029L;
+        long largeB = 16535734030L;
```

Comment:
would you mind making these big numbers explicitly Integer.MAX_VALUE and then add some constant?